### ISSUE TITLE
Uploading and Downloading the file of users

### ISSUE THREAD

**Issue by tealeafnguyen** [TeamNotSoBad/demo-web-project#1]
As of now, I am trying to use amazon web services S3C to upload the file of users to a bucket under my account as well as downloading.
The issues is that I think I should distribute keys for everyone to use so they can use my bucket, but that sounds like a major hassle, if anyone has a been solution for uploading and downloading the file of users, please let me know. Or if you can create an actual database that would be good too because I am having trouble wrapping my head around aws. 

TL;DR version: Should we upload and download the file of users instead of using an actual database?
If so, what should we use, because I'm not sure if aws is working properly for me and I might have to give everyone keys.

**Comment by tealeafnguyen** [comment]
UPDATE: Got the credentials working and amazon s3 will upload to my specified bucket when using a test program. Now I just need to figure out which request method would work to execute uploading and downloading the user map.
